src/main/java/teammates/logic/core/StudentsLogic.java

```diff
@@ -628,22 +620,32 @@ private StudentEnrollDetails enrollStudent(StudentAttributes validStudentAttribu
         return studentList;
     }
 
-    private List<String> getInvalidityInfoInDuplicatedEmail(String email,
-            List<String> studentEmailList, String[] linesArray) {
-        List<String> info = new ArrayList<>();
-        info.add("Same email address as the student in line \"" + linesArray[studentEmailList.indexOf(email) + 1] + "\"");
-        return info;
+    private String getInvalidityInfoInDuplicatedEmail(String[] linesArray, int duplicateEmailIndex) {
```

Comment:
inline this method

Reply:
How inline method is written ? All in one line ? :sweat_smile:

Reply:
Are you using IntelliJ? You can click on refactor -> inline and see what happens :P